### PR TITLE
docs: guide users to the right locations for questions

### PR DESCRIPTION
**Pull request by daviddias**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,13 +13,15 @@
...
-IPFS is a global, versioned, peer-to-peer filesystem. It combines good ideas from Git, BitTorrent, Kademlia, SFS, and the Web. It is like a single bittorrent swarm, exchanging git objects. IPFS provides an interface as simple as the HTTP web, but with permanence built in. You can also mount the world at /ipfs.
+IPFS is a global, versioned, peer-to-peer filesystem. It combines good ideas from previous systems such Git, BitTorrent, Kademlia, SFS, and the Web. It is like a single bittorrent swarm, exchanging git objects. IPFS provides an interface as simple as the HTTP web, but with permanence built in. You can also mount the world at /ipfs.
 
-For more info see: https://github.com/ipfs/ipfs.
+For more info see: https://docs.ipfs.io/introduction/overview/
 
-Please put all issues regarding:
-  - IPFS _design_ in the [ipfs repo issues](https://github.com/ipfs/ipfs/issues).
-  - Go IPFS _implementation_ in [this repo](https://github.com/ipfs/go-ipfs/issues).
+Before opening an issue, consider using one of the following locations to ensure you are opening your thread in the right location:
```
Comment (ericronne) [inline README.md]: I'd swap in "place" for the second "location," to avoid redundancy.

File: README.md [hunk 2]
```diff
@@ -13,13 +13,15 @@
...
+  - go-ipfs _implementation_ in [this repo](https://github.com/ipfs/go-ipfs/issues).
+  - IPFS _design_ in the [ipfs/specs issues](https://github.com/ipfs/specs/issues).
+  - Exploration of new ideas in the [ipfs/notes issues](https://github.com/ipfs/notes/issues).
```
Comment (ericronne) [inline README.md]: Delete "the"

File: README.md [hunk 3]
```diff
@@ -13,13 +13,16 @@
...
-IPFS is a global, versioned, peer-to-peer filesystem. It combines good ideas from Git, BitTorrent, Kademlia, SFS, and the Web. It is like a single bittorrent swarm, exchanging git objects. IPFS provides an interface as simple as the HTTP web, but with permanence built in. You can also mount the world at /ipfs.
+IPFS is a global, versioned, peer-to-peer filesystem. It combines good ideas from previous systems such Git, BitTorrent, Kademlia, SFS, and the Web. It is like a single bittorrent swarm, exchanging git objects. IPFS provides an interface as simple as the HTTP web, but with permanence built in. You can also mount the world at /ipfs.
 
-For more info see: https://github.com/ipfs/ipfs.
+For more info see: https://docs.ipfs.io/introduction/overview/
 
-Please put all issues regarding:
-  - IPFS _design_ in the [ipfs repo issues](https://github.com/ipfs/ipfs/issues).
-  - Go IPFS _implementation_ in [this repo](https://github.com/ipfs/go-ipfs/issues).
+Before opening an issue, consider using one of the following locations to ensure you are opening your thread in the right location:
```
Comment (ericronne) [inline README.md]:
```suggestion
Before opening an issue, consider using one of the following locations to ensure you are opening your thread in the right place:
```

File: README.md [hunk 4]
```diff
@@ -13,13 +13,16 @@
...
 
-For more info see: https://github.com/ipfs/ipfs.
+For more info see: https://docs.ipfs.io/introduction/overview/
 
-Please put all issues regarding:
-  - IPFS _design_ in the [ipfs repo issues](https://github.com/ipfs/ipfs/issues).
-  - Go IPFS _implementation_ in [this repo](https://github.com/ipfs/go-ipfs/issues).
+Before opening an issue, consider using one of the following locations to ensure you are opening your thread in the right place:
+  - go-ipfs _implementation_ bugs in [this repo](https://github.com/ipfs/go-ipfs/issues).
+  - Documentation issues in the [ipfs/docs issues](https://github.com/ipfs/docs/issues).
```
Comment (daviddias) [inline README.md]:
```suggestion
  - Documentation issues in [ipfs/docs issues](https://github.com/ipfs/docs/issues).
```

File: README.md [hunk 5]
```diff
@@ -13,13 +13,16 @@
...
+  - IPFS _design_ in the [ipfs/specs issues](https://github.com/ipfs/specs/issues).
```
Comment (daviddias) [inline README.md]:
```suggestion
  - IPFS _design_ in [ipfs/specs issues](https://github.com/ipfs/specs/issues).
```

File: README.md [hunk 6]
```diff
@@ -13,13 +13,16 @@
...
+  - Exploration of new ideas in the [ipfs/notes issues](https://github.com/ipfs/notes/issues).
```
Comment (daviddias) [inline README.md]:
```suggestion
  - Exploration of new ideas in [ipfs/notes issues](https://github.com/ipfs/notes/issues).
```

**Comment by ericronne** [comment]
🎉

**Pull request merged**